scripts/sanity_chk/sanitylib.py

```diff
@@ -2562,7 +2566,7 @@ def summary(self, unrecognized_sections):
                               str(instance.metrics.get("unrecognized", []))))
                 failed += 1
 
-            if instance.metrics.get('handler_time', None):
+            if instance.metrics.get('handler_time', None) and instance.status != "skipped" :
```

Comment:
how is this related and why it is needed?

Reply:
removed this since it not related with pytest function.